app/src/main/java/com/yeonae/chamelezone/view/login/FindEmailFragment.kt

```diff
@@ -34,7 +36,8 @@ class FindEmailFragment : Fragment() {
     private fun emailCheck(name: String, phone: String) {
         if (name == testName) {
             if (phone == testPhone) {
-                (activity as LoginActivity).replace(ConfirmEmailFragment().newInstance(
+                (activity as LoginActivity).replace(
+                    ConfirmEmailFragment().newInstance(
```

Comment:
```suggestion
                    ConfirmEmailFragment.newInstance(
```
ConfirmEmailFragment의 newInstance함수를 companion object에 만들어서 static으로 사용하는 패턴입니다.

Reply:
수정했습니다